dbms/src/AggregateFunctions/AggregateFunctionMinMaxAny.h

```diff
@@ -594,6 +600,10 @@ struct AggregateFunctionMinData : Data
     }
     bool changeIfBetter(const Self & to, Arena * arena) { return this->changeIfLess(to, arena); }
 
+    void prepareWindow() { throw Exception("Not implemented yet"); }
```

Comment:
Looks like `SingleValueDataGeneric` derived from `CommonImpl`, so `prepareWindow` is not needed here?

Reply:
> Looks like `SingleValueDataGeneric` derived from `CommonImpl`, so `prepareWindow` is not needed here?

done